src/js/components/Pagination/Pagination.js

```diff
@@ -96,7 +96,24 @@ const Pagination = forwardRef(
      * clickable index, control, or placeholder (e.g. ellipsis) indicating
      * more pages are available.
      */
-    const handleClick = event => {
+
+    const getItemIndices = nextPage => {
```

Comment:
```suggestion
    const getItemIndices = nextPage => {
```
maybe call it `getDataIndices` ?

Reply:
or maybe not since we've stuck with `numItems` as a prop.

Reply:
I am fine with that

Reply:
NVM - stick with item